### PR TITLE
docs about final and non-final rules

### PR DESCRIPTION
**Pull request by NickIliev**
Improved content and screenshots related to Rules + updated article about conditions, actions and final vs non-final actions/rules.

related to 
https://kinvey.atlassian.net/browse/FID-2149
https://kinvey.atlassian.net/browse/FID-6065 
https://kinvey.atlassian.net/browse/FID-1379 (support for importing SAZ in Rules)

### CODE REVIEW

File: collaboration/sharing-data.md [hunk 1]
```diff
@@ -137,6 +137,6 @@ The Fiddler Everywhere client informs the recipients about shared resources (inc
 
     * Confirm the opening of the resource.
 
-    * The shared resources load in the respective section (for example, a shared session loads in the **Snapshots** list, a rule in the **Rules Queue**, and a request in the **Requests** collection). If the shared resource is a snapshot of sessions, its content immediately loads in a new tab.
+    * The shared resources load in the respective section (for example, a shared session loads in the **Snapshots** list, a rule in the **Rules** queue, and a request in the **Requests** collection). If the shared resource is a snapshot of sessions, its content immediately loads in a new tab.
```
Comment (vukyarska) [inline collaboration/sharing-data.md:140]: 'a snapshot of sessions' - this one confuses me. maybe we can just keep **If the shared resource is a snapshot**?
Maybe the fact that we just renamed sessions to snapshots is overwhelming for me.

File: knowledge-base/how-to-use-the-rules-presets.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+But first, let us make a quick recap of what a Fiddler rule stands for:
+
+>tip A rule in Fiddler Everywhere contains [**a matching condition**]({%slug fiddler-rules-actions%}#conditions) (or set of conditions) that determines the target HTTP sessions and [**an action**]({%slug fiddler-rules-actions%}#actions) (or set of actions) that applies to the matched HTTP sessions.
+
+Fiddler provides multiple matching conditions related to HTTPS traffic (like protocols, schemes, timings, request and response data, and many others) to help you target a specific set of sessions. Then, the matched sessions are modified through the specified actions. As a result, you can quickly create logic that modifies the traffic without changing the client or the server application (or even having access to these applications). That said, the set of predefined rules that we created can be separated into several contextual groups based on what they are meant to achieve.
```
Comment (SvetlaGeorgieva) [inline knowledge-base/how-to-use-the-rules-presets.md:22]:
```suggestion
Fiddler provides multiple matching conditions related to HTTPS traffic (like protocols, schemes, timings, request and response data, and many others) to help you target a specific set of sessions. Then, the matched sessions are modified through the specified actions. As a result, you can quickly create logic that modifies the traffic without changing the client or the server application (or even without having access to these applications). That said, the set of predefined rules that we created can be separated into several contextual groups based on what they are meant to achieve.
```

File: knowledge-base/how-to-use-the-rules-presets.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+ com\.apple\.webkit|chrome|msedge
+ ```
+
+1. Create a **Do Not Show** action.
+
+    >important The **Do Not Show** action is final. No other action or active rule placed lower in the rules queue will be executed.
+
+This sample Fiddler rule hides all sessions except those from a Chromium-based browser like Microsoft Edge or Google Chrome.
+
+![Creating "Show Only Traffic from Specific Processes" rule](../images/advanced/adv-show-only-specific-processes.png)
```
Comment (SvetlaGeorgieva) [inline knowledge-base/how-to-use-the-rules-presets.md:82]: this image is showing the old rules dialog

File: modify-and-filter-traffic/conditions-and-actions.md [hunk 4]
```diff
@@ -433,52 +436,49 @@ Apart from returning files or predefined responses, a rule can perform the follo
...
-* Final actions are valid only when the rule is matched during the HTTP request.
-
-* If a session is matched with conditions that depend on its response (for example, a response body contains "HTML"), then any final action in any rule that matches the session will be ignored. The reason for this behavior is that final actions replace the response. By design, Fiddler is not intended to replace a response that was already received and matched conditions in a rule.
-
-* Non-final actions will allow other actions from the same rule or different rules with lower priority, which match the session, to execute.
-
+* Final actions prevent any other action from executing if this action comes lower on the list for the specific rule. 
 
-The following examples demonstrate what will happen when you combine final and non-final actions in one or multiple rules.
+* Final actions prevent any other rule from executing if this rule comes lower on the list of activated rules.
```
Comment (SvetlaGeorgieva) [inline modify-and-filter-traffic/conditions-and-actions.md:452]: "if this rule" -> it is unclear whether you speak of the final rule, or of the other rule that comes lower in the rules queue.

File: modify-and-filter-traffic/mocking-traffic-with-rules.md [hunk 5]
```diff
@@ -1,36 +1,46 @@
...
 
 ## Creating and Using a "Mock Traffic" Rule
 
 Fiddler can mock HTTPS traffic captured locally or remotely, for example, as traffic captured by another instance of Fiddler Everywhere and shared as a cloud session or SAZ archive. The following steps showcase how to use a SAZ archive to load and mock the captured traffic (in this case, as loaded from [https://example.com)](https://example.com)).
 
 
-1. Load the SAZ archive with the captured session snapshot. For demonstration purposes, the SAZ will contain traffic as captured from [https://example.com)](https://example.com).
+1. Capture the targeted HTTPS traffic or import an SAZ archive (containing the session snapshot) in the Snapshots list.
```
Comment (SvetlaGeorgieva) [inline modify-and-filter-traffic/mocking-traffic-with-rules.md:19]:
```suggestion
1. Capture the targeted HTTPS traffic or import a SAZ archive (containing the sessions snapshot) in the Snapshots list.
```

**Review by SvetlaGeorgieva — APPROVED** [review]

File: collaboration/sharing-data.md (hunk 1 above)
Comment (NickIliev) [reply collaboration/sharing-data.md:140]: revamped a bit

File: knowledge-base/how-to-use-the-rules-presets.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
+---
+title: HTTPS Traffic Modifications with Fiddler rules
+description: "Learn how to use Fiddler Everywhere rules for debugging, modifying, mocking, and testing the ongoing web traffic."
+type: how-to
+slug: how-to-use-fiddler-rules-to-modify-traffic
+publish: true
+res_type: kb
+---
+
+At one point or another, we need to test web applications and their behavior online. Whether we are talking about a tiny front-end change in a CSS (Cascading Style Sheets) selector, reproducing a reported scenario, or a significant update of the backend API, testing online services can take much work. It often requires creating safe staging environments (so your test keeps the live production jobs untouched) and re-building and re-deploying through time and money-consuming processes. The Fiddler's Rules functionality allows you to execute your tasks quickly and effeminately through the man-in-the-middle approach.
```
[the remark this answers is not in the document]
Comment (NickIliev) [reply knowledge-base/how-to-use-the-rules-presets.md:10]: nice catch :D

File: knowledge-base/how-to-use-the-rules-presets.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+We already discussed the **Rules** tab itself. This article will introduce you to a set of predefined rules that the team created as an entry point to display the power of the Rules in Fiddler Everywhere.
+
+## Rules Presets
+
+While the **Rules** tab allows users to create rules based on[ specific matching conditions and apply various actions]({%slug fiddler-rules-actions%}), the community often requires assistance in making rules. A further analysis showed that some Fiddler users needed to be made aware of the full capabilities of the Fiddler rules. For that reason, the team developed [a set of predefined rules]({%slug adv_techniques_fiddler%}) that you can quickly reuse with few clicks or use as a reference for your specific testing requirements.
```
[the remark this answers is not in the document]
Comment (NickIliev) [reply knowledge-base/how-to-use-the-rules-presets.md:16]: Yep, those work only after the docs build reads the yaml file and the internal links

**Pull request merged**